syntax = "proto3";

// Not necessary for Python, should still be declared to avoid name collisions
// in the Protocol Buffers namespace and non-Python languages
package compas_pb.data;

message PointData {
  // no inheritance, so Data will have to repeate but apparently
  string guid = 1;
  string name = 2;
  float x = 3;
  float y = 4;
  float z = 5;
}
